source/localizable/tutorial/routes-and-templates.md

```diff
@@ -218,21 +213,12 @@ Let's update our `index.hbs` with some HTML for our home page and our links to t
 
 In addition to providing button-style links in each route of our application, we would like to provide a common banner to display both the title of our application, as well as its main pages.
 
-When you create an Ember application with Ember CLI as we did, it generates a template called `application.hbs`.
-Anything you put in this template is shown for every page in the application.
-The default `application.hbs` file contains an `h2` tag with the text "Welcome to Ember", and an [`{{outlet}}`](http://emberjs.com/api/classes/Ember.Templates.helpers.html#method_outlet).
-The `{{outlet}}` defers to the router, which will render in its place the markup for the current route.
-
-```app/templates/application.hbs
-<h2 id="title">Welcome to Ember</h2>
+First, create a new file called `app/templates/application.hbs`.
```

Comment:
Sure. Will update.

On Fri, Jun 24, 2016 at 4:23 PM David Baker <notifications@github.com>
wrote:

> In source/localizable/tutorial/routes-and-templates.md
> <https://github.com/emberjs/guides/pull/1533#discussion_r68457000>:
>
> > @@ -218,21 +213,12 @@ Let's update our `index.hbs` with some HTML for our home page and our links to t
> >
> >  In addition to providing button-style links in each route of our application, we would like to provide a common banner to display both the title of our application, as well as its main pages.
> >
> > -When you create an Ember application with Ember CLI as we did, it generates a template called `application.hbs`.
> > -Anything you put in this template is shown for every page in the application.
> > -The default `application.hbs` file contains an `h2` tag with the text "Welcome to Ember", and an [`{{outlet}}`](http://emberjs.com/api/classes/Ember.Templates.helpers.html#method_outlet).
> > -The `{{outlet}}` defers to the router, which will render in its place the markup for the current route.
> > -
> > -```app/templates/application.hbs
> > -<h2 id="title">Welcome to Ember</h2>
> > +First, create a new file called `app/templates/application.hbs`.
>
> @toddjordan <https://github.com/toddjordan> We encouraged folks to use
> the ember g template application approach in the quick start, can we do
> that here?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/emberjs/guides/pull/1533/files/e97ff35628636bbc1b0d039a926fd2935c5c0941#r68457000>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADcR56Rn0vwkNApz9E9D4H4-x07NWVtIks5qPDyfgaJpZM4I9-t->
> .
>